lib/job-simulation.rb

```diff
@@ -6,9 +6,37 @@ class JobSimulation
 
   def initialize (jobs_available, job_seekers)
 
+      @workers = Stack.new
+      @waiting = Queue.new
+
+      job_seekers.times do |worker|
+        #establish initial Employed list (Stack)
+        if worker < jobs_available
+          @workers.push("Worker #{worker + 1}")
+        else
+          #establish initial Waitlist (Queue)
+          @waiting.enqueue("Worker #{worker + 1}")
+        end
+      end
   end
 
   def cycle
+    # roll a die
+    fired = rand(1..6)
```

Comment:
This should be `@roll = ` not `fired = `. If I'd written my tests that would be clear. But I didn't...